frontend/assets/translations/es.json

```diff
@@ -0,0 +1,921 @@
+            "use":"Use",
+            "preview":"Preview",
+            "leadGeneretion":"Lead generetion"
+        },
+        "templateLibraryModal":{
+            "select":"Select template",
+            "createAppfromTemplate":"Create application from template"
+        }
+    },
+    "confirmationPage":{
+        "setupAccount":"Set up your account",
+        "signupWithGoogle":"Sign up with Google",
+        "signupWithGitHub":"Sign up with GitHub",
+        "or":"OR",
+        "firstName":"First name",
+        "lastName":"Last name",
+        "company":"Company",
+        "role":"Role",
+        "pleaseSelect":"Please select",
+        "password":"Password",
+        "confirmPassword":"Confirm Password",
+        "clickAndAgree":"By clicking the button below, you agree to our",
+        "termsAndConditions":"Terms and Conditions",
+        "finishAccountSetup":"Finish account setup",
+        "acceptInvite":"accept invite",
+        "accountExists":"Already have an account?",
+        "and":"and"
+
+    },
+    "onBoarding":{
+        "finishToolJetInstallation":"Finish ToolJet installation",
+        "organization":"Organization",
+        "name":"Name",
+        "email":"Email",
+        "receiveUpdatesFromToolJet":"You will receive updates from the ToolJet team ( 1-2 emails every month, we do not spam )",
+        "finishSetup":"Finish setup",
+        "skip":"Skip"
+    },
+    "redirectSso":{
+        "upgradingTov1.13.0":"Upgrading to v1.13.0 and above.",
+        "fromV1.13.0":"From v1.13.0 we have introduced",
+        "multiWorkspace":"Multi-Workspace",
+        "singleSignOnConfig":"The Single Sign-On related configurations are moved from environment variables to database. Please refer this",
+        "link":"Link",
+        "toConfigureSSO":"to configure SSO.",
+        "haveGoogleGithubSSo":"If you have Google or GitHub SSO configurations before upgrade and disabled Multi-Workspace, then theSSO configurations will be migrated while upgrade but you have to re-configure the redirect URL in the SSO provider side. Redirect URLs for each SSO are given below.",
+        "isMultiWorkspaceEnabled":"If you have enabled Multi-Workspace, then the SSO configurations will not be migrated while upgrade so you have to re-configure the SSO under the respective workspace.",
+        "youHaveEnabled":"You have Enabled",
+        "setupSsoWorkspace":"Please login with password and you can setup sso using workspace",
+        "manageSsoMenu":"Manage SSO menu.",
+        "youHaveDisabled":"You have Disabled",
+        "configureRedirectUrl":"Please configure redirect url in SSO provider side.",
+        "google":"Google",
+        "redirectUrl":"Redirect URL:",
+        "gitHub":"GitHub"
+    },
+    "oAuth2": {
+        "pleaseWait":"Please wait...",
+        "authSuccess":"Auth successful, you can close this tab now.",
+        "authFailed":"Auth failed"
+    },
+    "widgetManager": {
+        "commonlyUsed": "comúnmente utilizado",
```

Comment:
Hi, just a small detail here.
In the value of each translation I would recommend to start with a capital letter, to maintain the consistency with the other values in the file.